gitlab/v4/objects.py

```diff
@@ -1425,9 +1437,22 @@ class GroupManager(CRUDMixin, RESTManager):
             "name",
             "path",
             "description",
+            "membership_lock",
+            "share_with_group_lock",
             "visibility",
+            "request_access_enabled",
+            "mentions_disabled",
```

Comment:
Thanks for having me take a look :D sorry this took so long!
It seems like there are some duplicate entries (I saw some duplicates even in the GitLab docs) that argpase and the CLI do not like - this should make it go green:
```suggestion
```

Reply:
Ah no problem. Sorry for assigning, if not even the pipeline is green 🙈